keyboards/keycapsss/3w6_2040/config.h

```diff
@@ -0,0 +1,15 @@
+// Copyright 2022 Conor Burns (@Conor-Burns)
+// Copyright 2023 Ben Roe (@keycapsss)
+// Copyright 2023 Tom Barnes (@keyboard-magpie)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN GP0
+#define SERIAL_USART_RX_PIN GP1
+#define SERIAL_USART_PIN_SWAP
+#define SERIAL_USART_SPEED 921600
+#define USB_VBUS_PIN GP19
```

Comment:
This can go to info.json.
```suggestion
```

Reply:
Its not currently available in `info.json`.

Reply:
Soon :tm: or something. Will resolve this.